### PR TITLE
[DOCS] Update Compare Two Tables with the Onboarding Data Assistant

### PR DESCRIPTION
**Pull request by kwcanuck**
In [issue 8320](https://github.com/great-expectations/great_expectations/issues/8320), a request was submitted to update the code sample in the _Profile Source Data_ section of _Compare two tables with the Onboarding Data Assistant_.

Resolves 8320

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

### CODE REVIEW

**Review by donaldheppner — APPROVED** [review]

**Review by alexsherstinsky — APPROVED** [review]

File: tests/integration/test_definitions/multiple_backend/integration_tests.py
Comment (Shinnnyshinshin) [inline tests/integration/test_definitions/multiple_backend/integration_tests.py:32]: Was this a stray comment?

Comment (alexsherstinsky) [reply tests/integration/test_definitions/multiple_backend/integration_tests.py:32]: @Shinnnyshinshin I did not enter this comment, but it seems appropriate to me; am I missing an error in it?  Thanks!

File: tests/integration/docusaurus/expectations/advanced/data_assistant_cross_table_comparison.py
Comment (Shinnnyshinshin) [inline tests/integration/docusaurus/expectations/advanced/data_assistant_cross_table_comparison.py:70]: Is there a reason we are changing the batch request from `pg` to `mysql`? Also, what was the reason for changing the `exclude_column_names`?

Comment (alexsherstinsky) [reply tests/integration/docusaurus/expectations/advanced/data_assistant_cross_table_comparison.py:70]: @Shinnnyshinshin Yes, the `pg` as a typo, which had to be corrected.  The `exclude_column_names` is needed because the numerical calculations between the two database backends are not the same.  Thank you.

Comment (austiezr) [reply tests/integration/docusaurus/expectations/advanced/data_assistant_cross_table_comparison.py:70]: We're dealing with 2 different batch requests, and one of them was used in the wrong place for the flow of the script.

Additionally, we're excluding some columns to avoid failing validations that were causing the script to fail.

File: tests/integration/test_definitions/multiple_backend/integration_tests.py
Comment (Shinnnyshinshin) [reply tests/integration/test_definitions/multiple_backend/integration_tests.py:32]: We have the instructions here: 
```
https://github.com/great-expectations/great_expectations/blob/7b4265942e3ce4e43799ccc5b4bc45db51605482/tests/integration/integration_test_fixture.py#L14
```
If we are adding this inline comment for this `IntegrationTestFixture` only, it feels out of place.

File: docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md]:
```suggestion
In the previous example, specific columns were excluded to prevent Expectations from being set against them.
```

File: docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md]:
```suggestion
In our hypothetical use case these inconsistencies are tolerated, and therefore Expectations are not set against the columns likely to generate the errors.
```

File: docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md]:
```suggestion
This is an example of how an Expectation Suite created by the Data Assistant can be customized.
```

File: docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md]:
```suggestion
Some SQL dialects handle data types in different ways, and this can cause precision mismatches on some numbers.
```

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Everything is great; just waiting for the release to finish; will unblock thereafter.  Thank you.

**Review by alexsherstinsky — APPROVED** [review]

**Pull request merged**
